src/Admin/Settings.php

```diff
@@ -337,6 +337,18 @@ public function triggersSettings($settings)
 					'render' => [new CoreFields\Checkbox(), 'input'],
 					'sanitize' => [new CoreFields\Checkbox(), 'sanitize'],
 				]
+			)
+			->addField(
```

Comment:
It just struck me, we don't really need that option here, it adds much complexity to the UX. Could you please remove really both `email_address_change_request` and `email_address_changed` options?